tests/buildenv.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -eu
+
+ERROR=1
+OK=0
+RESULT="${OK}"
+
+set_error()
+{
+    RESULT="${ERROR}"
+}
+
+are_all_build_env_preconditions_met()
```

Comment:
Clearly not a fan of writing things out.. :-) 
I can argue that this is both descriptive an KISS. Yest it makes the script a few bytes larger and yes if you are working in VGA mode or something it could be more difficult to read. 
To build your case i was more thinking in the line of .. that it would me nice to separate tests in clearly defined functions, then we can add more buildenv test functions later. But YAGNI kicks in and therefore i might as well remove it completely from the function at this point.

Reply:
my dilexia doesn't go well with writing things out too long :) also human brains don't like long things generally :p but yeah, the name doesn't bother me THAT much, I was just curious :)